### ISSUE TITLE
Error after fresh install on Ubuntu 22.04

### ISSUE THREAD

**Issue by kurttheviking** [AtomLinter/linter-eslint-node#25]
## Issue Type
Bug

## Issue Description
Just installed Ubuntu 22.04 and reinstalling everything, including Atom. The first package I installed was linter-eslint for v7+. It seems the linter is unable to start and immediately throws `getRulesMetaForResults is not a function`.

## Bug Checklist
- [ x] Restart Atom
- [ x] Verify the `eslint` CLI gives the proper result, while `linter-eslint-node` does not
- [ ] Paste the output of the `Linter Eslint Node: Debug` command from the Command Palette below

```
n/a
```

**Comment by savetheclocktower** [comment]
Does the “Linter Eslint Node: Debug” command not execute at all? The error message you posted is one that would be thrown when the package tries to lint a file, not beforehand. It will try to lint JavaScript files (in ESLint projects) if such a project is restored from a previous session on startup. If it gets that far, the package has initialized successfully.

That error message also suggests a situation where the package thinks it's successfully found a version of ESLint to load, but is wrong. I'm curious how the project in question is structured, and whether ESLint is installed globally or locally.

**Comment by kurttheviking** [comment]
The error was definitely thrown when it tried to lint a file but I wasn't able to run the debug command. I tried both immediately after a fresh install and after a Atom restart as well as after a full system restart, just to see if that would make a difference. In any case eslint was installed locally (`7.22.0`) &ndash; I was in a hurry to reset everything after the Ubuntu upgrade so I simply downgraded to `linter-eslint@9.0.1` and everything seemed to be working as expected again (files linting on save, etc.) So I haven't bothered to go back and try again with `linter-eslint-node`.
